.github/workflows/latest-spring.yml

```
@@ -24,7 +24,7 @@ jobs:
     name: Java ${{ matrix.java }}
     strategy:
       matrix:
-        java: [8, 11]
+        java: [11, 17]
```

The latest v3 version is going to require 17+, my suggestion is to leave the matrix array, as we may need want to add newer versions like 21 in the future


```suggestion
        java: [17]
```